tensorrt_llm/_torch/auto_deploy/transform/library/mlir_elementwise_fusion.py

```
@@ -0,0 +1,164 @@
+        num_skipped = 0
+        for sg in subgraphs:
+            if _max_input_rank(sg) < 2 or _min_output_rank(sg) < 2:
+                num_skipped += 1
+                continue
+            try:
+                kernel_fn = generate_kernel_from_subgraph(sg)
+                if kernel_fn is not None:
+                    sg_hash = KernelCache.hash_subgraph(sg)
+                    replace_subgraph_with_fused_op(sg, kernel_fn, sg_hash, converter.metadata)
+                    num_replaced += 1
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)